ldapadmin/src/main/webapp/console/app/assets/index.html

```diff
@@ -8,11 +8,14 @@
   <link rel="stylesheet" href="public/app.css">
 
   <title>Admin Console</title>
+
+  <script src="/header/js/header.js"></script>
```

Comment:
should not be required anymore since https://github.com/georchestra/georchestra/commit/15ed5627b6cfa47eb7375825fdf78db1e2325abd merged into 15.12 and master